apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestSchemaManager.java

```diff
@@ -93,7 +117,7 @@ private static Map<String, String> createScriptPlaceholdersMap(DataSource dataSo
         placeholders.put("logic.perms", "--");
         placeholders.put("logic.psql-only", "-- ");
         placeholders.put("logic.hsql-only", "");
-      } else if (DatabaseUtils.isPostgresConnection(connection)) {
+      } else if (connection.getMetaData().getDatabaseProductName().equals("PostgreSQL")) {
```

Comment:
any particular reason to move away from the helper methods here and below in the exception?

Reply:
DatabaseUtils is a utility within the bfd-shared-utils package, and bfd-shared-test-utils is a dependency of shared-utils, so nothing in that can be used here to avoid circular dependencies. This was one of the few places shared-utils had a util class used from it in the test code, so I just copied the small code snippet needed in this instance.